tests/ui/same_name_method.rs

```diff
@@ -62,7 +62,7 @@ mod should_lint {
         impl T1 for S {}
     }
 
-    mod multiply_conflicit_trait {
+    mod multiply_conflict_trait {
```

Comment:
?

Reply:
Should be 'multiple_conflicting_traits'.